students/MichaelM/lesson2/Lesson2_Ex2_FizzBuzz.py

```diff
@@ -0,0 +1,30 @@
+# cd C:\Users\v-micmcd.Redmond\Dropbox\UW_Python\git\Self_Paced-Online\students\MichaelM\lesson2
+# cd C:\Users\geekw\Dropbox\UW_Python\git\Self_Paced-Online\students\MichaelM\lesson2
+# run Lesson2_Ex2_FizzBuzz.py
+# git status
+# git add Lesson2_Ex2_FizzBuzz.py
+# git commit
+# git push
+# goto https://github.com/geekwriter2/Self_Paced-Online/tree/master/students
+# click Pull request > new pull request
+# import name_main
+# run Lesson2_Ex1.py
+# from Natasha: python uses snake_case convention for var names (and functions)> horizontal_line
+
+def buzzles_and_fizzles(top_pot):
+        for i in range(top_pot):
```

Comment:
another thing to keep in mind is indentation, in python it's pretty important and convention is to use 4 spaces (not tabs).

if you use an editor like sublime, you can configure or Pycharm IDE does it out of the box.